### DISCUSSION TITLE
How do I setup Nuxt Three and Jest?

### DISCUSSION THREAD

**Discussion by jtrimm007** [nuxt/nuxt#20612] [Questions]
I am currently trying to use Nuxt 3 and Jest in a project. I have an issue where Jest cannot find the auto import of useNuxtApp and I'm not sure what to do... Is there an example out there someone could point me towards?

**Comment by warflash** [comment]
Have you checked out whether the setup mentioned in the docs works for you?
> In Nuxt 3, we have a rewritten version of @nuxt/test-utils. We support [Vitest](https://github.com/vitest-dev/vitest) and [Jest](https://jestjs.io/) as test runners.

https://nuxt.com/docs/getting-started/testing#setup
